webapi/src/main/scala/org/knora/webapi/responders/v2/SearchUtil.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2021 - 2023 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.responders.v2
+
+import org.knora.webapi.IRI
+import org.knora.webapi.store.triplestore.api.TriplestoreService.Queries.Construct
+import org.knora.webapi.store.triplestore.api.TriplestoreService.Queries.Select
+
+object SearchUtil {
```

Comment:
How about renaming this to `SearchQueries` which would reveal the intention of this "util" more.

Reply:
you're right, changed it accordingly